### PR TITLE
Multilingual

### PR DESCRIPTION
**Pull request by andyofsmeg**
New page with information about partner initiatives from R Consortium etc.

### CODE REVIEW

File: content/partners.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+## Clinical Statistical Reporting in a Multilingual World
+
+A Phuse initiative called, ['Clinical Statistical Reporting in a Multilingual World'](https://github.com/phuse-org/CSRMLW) started in 2020.  As with the R Consortium initiatives, this Phuse initiative maintains a close connection with the R Validation Hub.  As they put it on their GitHub page, "Subtle differences exist between the fundamental approaches implemented by each language, yielding differences in results which are each correct in their own right."  One of the specific aims of this initiative is to "provide a framework for assessing the fundamental differences for a particular statistical analysis across languages". 
+
+
+# <img src="/partners_files/psi.png", height=50, alt='PSI'> 
+
+## AIMS SIG
+
+The R Validation Hub was started by the [Application and Implementation of Methodologies in Statistics (AIMS) Special Interest Group (SIG)](https://psiweb.org/sigs-special-interest-groups/aims) within PSI.  Supporting the R Validation Hub is one of the SIG's stated objectives along with bridging the gap to SAS and aducating members in R more broadly.
```
Comment (marlycormar) [inline content/partners.md:36]: Small typo: *educating* rather that aducating

File: content/partners.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Partner Initiatives"
+---
+
+# <img src="/partners_files/rconsort.png", height=50, alt='R Consortium'>
```
Comment (marlycormar) [inline content/partners.md:5]: Image doesn't display

**Comment by marlycormar** [comment]
Hi @andyofsmeg, the PR looks good overall. I added some comments above.

**Comment by jmanitz** [comment]
Important contribution. Looks good to me. Only a minor remark: "R/pharma in the Boston area" (Cambridge is its own city)

File: content/partners.md (hunk 1 above)
Comment (jmanitz) [reply content/partners.md:36]: Small correction:  R/Pharma Conference in Cambridge, Boston area.

**Review by jmanitz — APPROVED** [review]

**Comment by joseph-rickert** [comment]
You could use some of the material from the READMEs for these groups
https://github.com/RConsortium/rtrs-wg

### R Tables for Regulatory Submission (RTRS)

The R Consortium working group, R Tables for Regulatory Submissions (RTRS) includes representation from several large pharmaceutical companies and CRO’s. The goal of the working group is to create standards for creating tables that meet the requirements of FDA submission documents, and hence enhance the suitability of R for FDA submissions. It is part of a larger R Consortium effort to facilitate the certification and validation of R packages and tools for FDA submissions thereby allowing drug developers to submit documentation for regulatory approval using the R programming environment in conjunction with open-source packages without the need for closed and often expensive proprietary tools.

### Submissions https://github.com/RConsortium/submissions-wg

R Submissions Working Group

The R Consortium "R Submissions" Working Group is focused on IT and platform challenges that must be addressed in order to make "all R" regulatory submissions.

The mission of the group partially overlaps with work underway in the R Validation Hub and RTRS working groups, and eventually these groups may see some consolidation. However, for now, we have the opportunity to set the agenda for this new area.

File: content/partners.md (hunk 2 above)
Comment (andyofsmeg) [reply content/partners.md:5]: Seems to work locally

File: content/partners.md (hunk 1 above)
Comment (andyofsmeg) [reply content/partners.md:36]: Corrections made, thanks.

**Pull request merged**